packages/dx-react-demos/src/demo-sources/grid-basic/static-column-width.jsxt

```diff
@@ -0,0 +1,51 @@
+import * as React from 'react';<%&additionalImports%>
+import {
+  Grid,
+  Table,
+  TableHeaderRow,
+} from '@devexpress/dx-react-grid-<%&themeName%>';
+
+import {
+  generateRows,
+  globalSalesValues,
+} from '../../../demo-data/generator';
+
+export default class Demo extends React.PureComponent {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      columns: [
+        { name: 'region', title: 'Region' },
+        { name: 'sector', title: 'Sector' },
+        { name: 'channel', title: 'Channel' },
+        { name: 'customer', title: 'Customer' },
+        { name: 'product', title: 'Product' },
+        { name: 'amount', title: 'Sale Amount' },
+      ],
+      rows: generateRows({ columnValues: globalSalesValues, length: 8 }),
+      tableColumnExtensions: [
+        { columnName: 'region', width: 180 },
+        { columnName: 'sector', width: 130 },
+        { columnName: 'customer', width: 240 },
+        { columnName: 'product', width: 210 },
+        { columnName: 'amount', width: 140 },
```

Comment:
Maybe we should display the `align` property too?

Reply:
I believe we should not mix two demos